src/url-replacements.js

```diff
@@ -144,6 +148,82 @@ class UrlReplacements {
     this.set_('SCREEN_HEIGHT', () => {
       return this.win_.screen.height;
     });
+
+    // Returns the time it took to load the whole page. (excludes amp-* elements
+    // that are not rendered by the system yet.)
+    this.set_('PAGE_LOAD_TIME', () => {
```

Comment:
I see. Yes, I think this is fine. As long as it's documented clearly. E.g. "use of this variable indicates that it will wait until the event has happened or such".